BotCommands/LoLCommands.cs

```diff
@@ -33,21 +37,44 @@ public async Task GetGameDataLoL(CommandContext ctx, string msg)
             {
                 if (player.Victory)
                     embed.AddField((player.Name), player.Champion, true);
-                else if (newLine) { 
-                    newLine = false; 
+                else if (newLine)
+                {
+                    newLine = false;
                     embed.AddField("--- Defeat", "---", false);
-                    }
+                }
                 if (!player.Victory)
                 {
                     embed.AddField((player.Name), player.Champion, true);
                 };
             }
-
             await ctx.RespondAsync(embed: embed.Build()).ConfigureAwait(false);
         }
 
-        
-        
-        
+        [Command("lolRotation")]
+        public async Task GetFreeRotationLoL(CommandContext ctx)
+        {
+
+            var tempConfigfromFileStream = string.Empty;
```

Comment:
This can be extracted to a separate method (for example LoadBotConfiguration() )